README.md

```diff
@@ -1 +1,39 @@
-# SAP2000_Toolkit
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/plmqiho414qw9oko/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/sap2000_toolkit/branch/master)
```

Comment:
Please include the Azure badge as available from DevOps.

Reply:
Oh, I copied from your comment in the issue. Where is DevOps?

Reply:
The comment on the issue was just for AppVeyor because they're not publicly available.
The DevOps badge is `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/SAP2000_Toolkit/SAP2000_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=202&branchName=master)`

Reply:
```suggestion
[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/plmqiho414qw9oko/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/sap2000_toolkit/branch/master)[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/SAP2000_Toolkit/SAP2000_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=202&branchName=master)
```

Reply:
Appending after the appveyor badge OK?

Reply:
Aye